tasks/cachebust.js

```diff
@@ -67,7 +67,10 @@ module.exports = function(grunt) {
             var isCSS = (/\.css$/).test(filepath);
 
             findStaticAssets(markup, isCSS).forEach(function(reference) {
-                var filePath = (opts.baseDir ? opts.baseDir : path.dirname(filepath)) + '/';
+                // original code (doesn't handle css in a subdirectory with relative paths to assets):
+                // var filePath = (opts.baseDir ? opts.baseDir : path.dirname(filepath)) + '/';
+                // simpler new code. filepath already includes opts.baseDir so no need to check again:
+                var filePath = path.dirname(filepath) + '/';
```

Comment:
This is the bit that worries me most. It made me aware that there is currently no test checking that the `baseDir` option is applied correctly. I've created a test that does just that over at #106. I have a feeling once you've updated this branch with that change it will start failing. I'll grab a copy of your branch and have a play around as well.

I've tried to avoid supporting relative paths in files because it's a bit of an annoyance really. It's always safer to give it a relative path from the root.